backend/Controllers/SightingsController.cs

```diff
@@ -30,4 +31,23 @@ public async Task<IActionResult> Create(SightingsRequest sightingRequest)
         }
     }
 
+    [HttpDelete("sighting={sightingId}&user={userId}/delete")]
```

Comment:
my understanding of the HttpDelete attribute here is that you don't need have 'delete' in the url as using a delete request is sufficient to ensure the right method is called.

if you did want to keep 'delete' in the url you would want it to be before the query parameters (e.g. 'delete/sighting={sightingId}&user={userId}') because query parameters should allways come at the end of the url.